src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double CAPACITY_INCREASE_MODIFICATOR = 1.5;
+    private Object[] defaultList;
```

Comment:
Our class is parameterized, so maybe we should declare defaultList as T[] array not the Object[]?
`private Object[] defaultList;`